protobuf3-cpp/plan.sh

```diff
@@ -0,0 +1,17 @@
+pkg_name=protobuf3-cpp
```

Comment:
This should just be named `protobuf-cpp`

Reply:
@reset sounds good to me. The only reason I did that is because there is already a `protobuf` plan and I wanted to make sure we'all know that the version 3.X actually started creating them for specific programming languages. But if you are onboard to call it straight up `protobuf-cpp`, I'm down with that! Let me change that really quick! 👍